Compiler.Lexer/Compiler.Lexer/Tokenizer.cs

```diff
@@ -0,0 +1,403 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Compiler.Lexer
+{
+    public partial class Tokenizer
```

Comment:
Should have "\\\" code comments for class and methods. Definitions, inputs, return values, assumptions and other relevant info for others looking at it, either in code or other developers using the class . Also, since it is a partial class it should explain what, why, how, expectations of the other partial implementation.